starboard/android/apk/app/src/main/java/dev/cobalt/media/AudioTrackBridge.java

```diff
@@ -165,7 +168,8 @@ public AudioTrackBridge(
       audioTrackBufferSize /= 2;
     }
     if (isAudioTrackValid()) {
-      audioTrackBufferSize = audioTrack.getBufferSizeInFrames() * channelCount * getBytesPerSample(sampleType);
+      audioTrackBufferSize =
+          audioTrack.getBufferSizeInFrames() * channelCount * getBytesPerSample(sampleType);
```

Comment:
No need to multiply by the channel count when calling `getBufferSizeInFrames()`, it already returns the size in bytes for compressed formats.

https://developer.android.com/reference/android/media/AudioTrack#getBufferSizeInFrames()

Reply:
Fixed.